ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -162,14 +165,28 @@ sFlow is disabled.
 
 | Interface | PVLAN Mapping | Secondary Trunk |
 | --------- | ------------- | ----------------|
+| Ethernet1 | 20-30 | True |
+| Ethernet2 | - | False |
 | Ethernet15 | 111 | - |
 | Ethernet17 | - | True |
 
 ##### VLAN Translations
 
-| Interface | From VLAN ID(s) | To VLAN ID | Direction |
-| --------- | --------------- | -----------| --------- |
-| Ethernet16 | 111-112 | 110 | out |
+| Interface | Direction | From VLAN ID(s) | To VLAN ID | From Inner VLAN ID | To Inner VLAN ID | Network | Dot1q-tunnel |
+| --------- | --------- | --------------- | ---------- | ------------------ | ---------------- | ------- | ------------ |
+| Ethernet1 | both | 12 | 20 | - | - | - | - |
+| Ethernet1 | both | 24 | 46 | 78 | - | True | - |
+| Ethernet1 | both | 24 | 46 | 78 | - | False | - |
+| Ethernet1 | both | 43 | 30 | - | - | - | True |
+| Ethernet1 | in | 10 | 24 | - | - | - | - |
+| Ethernet1 | in | 23 | 45 | - | - | - | True |
+| Ethernet1 | in | 37 | 49 | 56 | - | - | - |
+| Ethernet1 | out | 10 | 45 | - | 34 | - | - |
+| Ethernet1 | out | 34 | 50 | - | - | - | - |
+| Ethernet1 | out | 45 | True | - | - | - | True |
```

Comment:
Why To VLAN ID is True when we did not provided as input?

Reply:
Fixed, added `dot1q_tunnel_to` key inside `direction_out`